.github/workflows/ruby.yml

```diff
@@ -81,6 +81,8 @@ jobs:
       PGPASSWORD: postgres
       RAILS_ENV: test
       JIRA_ENVIRONMENT_FIELD: customfield_10000
+      SENDGRID_USERNAME: metrics@rootstrap.com
```

Comment:
given that this repo is public maybe keep `rootstrap` name out, use `example.com` instead